mineinabyss-features/src/main/kotlin/com/mineinabyss/helpers/CosmeticHelpers.kt

```diff
@@ -0,0 +1,41 @@
+package com.mineinabyss.helpers
+
+import com.mineinabyss.components.cosmetics.cosmetics
+import com.mineinabyss.components.playerData
+import com.mineinabyss.mineinabyss.core.mcCosmetics
+import io.lumine.cosmetics.api.cosmetics.ItemCosmetic
+import io.lumine.cosmetics.managers.back.BackAccessory
+import io.lumine.cosmetics.managers.gestures.GestureManager
+import io.lumine.cosmetics.managers.hats.Hat
+import io.lumine.cosmetics.players.Profile
+import org.bukkit.Material
+import org.bukkit.entity.Player
+import org.bukkit.inventory.ItemStack
+
+fun Player.playGesture(gesture: String) {
+    mcCosmetics.profiles.getProfile(name).ifPresent { profile: Profile? ->
```

Comment:
You could probably do something like `...getOrNull() ?: return` (not sure tho) but it might look a bit nicer